qemu/tests/cfg/qemu_guest_agent_hotplug.cfg

```diff
@@ -1,6 +1,6 @@
 - qemu_guest_agent_hotplug: install setup image_copy unattended_install.cdrom single_driver_install.with_vioserial
     only Fedora, RHEL, Windows
-    only x86_64,i386,ppc64,ppc64le
+    only x86_64,i386,ppc64,ppc64le,s390x,aarch64
```

Comment:
Can safely remove this line directly I think.

Reply:
Got it,  done